### PR TITLE
Warning and changelog link fixes

### PR DESCRIPTION
**Pull request by neon60**
The incorrect `.gitignore` caused trouble with an old CHANGELOG.md. I also fixed the internal links to work at two separate location.

### CODE REVIEW

**Comment by neon60** [comment]
@samjwu Can you check this PR?

File: docs/conf.py [hunk 1]
```diff
@@ -85,7 +85,7 @@
     {"file":"about/release-notes", "os":["linux"]},
 ]
 
-exclude_patterns = ['temp']
+exclude_patterns = ['temp', 'release/versions.md']
```
Comment (samjwu) [inline docs/conf.py:88]: I'm not sure if we want to exclude this. There's no explicit link in the repo, but the header includes it via rocm-docs-core (top right Version List)

File: tools/autotag/templates/rocm_changes/5.5.0.md [hunk 2]
```diff
@@ -25,8 +25,8 @@ The following hipcc changes are implemented in this release:
   `hipcc` package for installing `hipcc` binaries in future ROCm releases.
 
 * In a future ROCm release, the following samples will be removed from the `hip-tests` project.
-  * `hipBusbandWidth` at <https://github.com/ROCm-Developer-Tools/hip-tests/tree/develop/samples/1_Utils/shipBusBandwidth>
-  * `hipCommander` at <https://github.com/ROCm-Developer-Tools/hip-tests/tree/develop/samples/1_Utils/hipCommander>
+  * `hipBusbandWidth` from <https://github.com/ROCm-Developer-Tools/hip-tests/tree/develop/samples/1_Utils/>
+  * `hipCommander` from <https://github.com/ROCm-Developer-Tools/hip-tests/tree/develop/samples/1_Utils/>
```
Comment (samjwu) [inline tools/autotag/templates/rocm_changes/5.5.0.md:29]:
```suggestion
  * `hipBusbandWidth` from <https://github.com/ROCm/hip-tests/tree/develop/samples/1_Utils/>
  * `hipCommander` from <https://github.com/ROCm/hip-tests/tree/develop/samples/1_Utils/>
```

**Review by samjwu — COMMENTED** [review]
looks good to me.
had 1 comment about the conf.py

File: docs/conf.py (hunk 1 above)
Comment (neon60) [reply docs/conf.py]: Removed now. I will ask the team about rocm-docs-core and will return to this.

**Comment by neon60** [comment]
@samjwu I can also put [this](https://github.com/ROCm/ROCm/pull/2853/commits/e2d55482989b198c4d70982f437f8c9e93004702) commit to separate PR. It's still related to the problem and your findings highlighted the problem

**Review by samjwu — APPROVED** [review]

**Review by Rmalavally — DISMISSED** [review]

**Comment by neon60** [comment]
Rebased the branch.

**Review by Rmalavally — APPROVED** [review]

**Review by samjwu — APPROVED** [review]

**Pull request merged**